ZividOpenCV/ZDF2OpenCV/ZDF2OpenCV.cpp

```diff
@@ -130,9 +130,9 @@ int main()
         cv::applyColorMap(z, zJetColorMap, cv::COLORMAP_JET);
 
         // Setting nans to black
-        for(int i = 0; i < pointCloud.height(); i++)
+        for(size_t i = 0; i < pointCloud.height(); i++)
         {
-            for(int j = 0; j < pointCloud.width(); j++)
+            for(size_t j = 0; j < pointCloud.width(); j++)
```

Comment:
Why not `auto`?

Reply:
Both works. Any reason to change it?

Reply:
In case whatever we are iterating over changes type?

Reply:
That could happen if Zivid breaks its api. I don't think it really matters, but have fixed it now.

Reply:
Wait, `auto` won't work. `auto j = 0` deduces `int`. We want `size_t`. Reverting my fix.